android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -297,6 +298,15 @@ public boolean setKeyboard(String keyboardID, String languageID) {
       Log.d("KMKeyboard", jsString);
     }
 
+    // Add a little delay for the keyboard to load so the chirality can be properly detected
+    Handler chiralityHandler = new Handler();
+    chiralityHandler.postDelayed(new Runnable() {
+      @Override
+      public void run() {
+        loadUrl("javascript:setIsChiral()");
+      }
+    }, 1000);
```

Comment:
This worries me a little bit: is there any way we can instead respond on a keyboardLoaded event from KMW instead within keyboard.html? I dislike using timeouts that assume that the keyboard is ready.